### ISSUE TITLE
@input event not triggered in Select component when using multiple selection

### ISSUE THREAD

**Issue by Vaerum** [quasarframework/quasar#395]
Hi
I cannot get the select component to fire the vue event @input.

I have tested the component and it does not fire the set function.
```
  computed: {
    model: {
      get () {
        if (this.multipleSelection && !Array.isArray(this.value)) {
          console.error('Select model needs to be an array when using multiple selection.')
        }
        return this.value
      },
      set (value) {
       // Does not fire
        this.$emit('input', value)
      }
    },
```

Can someone see the problem?

Best regards
Rasmus

**Comment by rstoenescu** [comment]
Will investigate and fix. You are probably working with multiple selection so your model is an array. Computed props are not deep watchers.
Workaround would be to make a deep watcher yourself on the scope variable used for `v-model` on Select.

**Comment by Vaerum** [comment]
@rstoenescu thank you. You are correct, it is an array. I will find an work around.

Would it be bad to place the $emit in the get()?

**Comment by rstoenescu** [comment]
Yes. Any rerender will emit @input then :) place a watcher in your app code like I mentioned until I got time to fix this.

**Comment by rstoenescu** [comment]
Fix available in future v0.14.
